packages/cli/examples/README.md

```
@@ -0,0 +1,306 @@
+
+## Web Examples
+
+### web-serve.json
+
+**Purpose**: Browser bundle that sends events to collector
+
+**Features**:
+
+- sourceDemo (generates test events automatically)
+- destinationDemo (console output for debugging)
+- destinationAPI (sends events to http://localhost:8080/collect)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)